src/app-account.js

```diff
@@ -38,8 +38,14 @@ app.post('/bootstrap', (req, res) => {
 });
 
 app.post('/login', (req, res) => {
-  let token = login(req.body.password);
-  res.send({ status: 'ok', data: { token } });
+  let { error, token } = login(req.body.password);
+
+  if (error) {
+    res.status(400).send({ status: 'error', reason: error });
+    return;
+  } else {
+    res.send({ status: 'ok', data: { token } });
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏unnecessary `else` statement

```suggestion

  res.send({ status: 'ok', data: { token } });
```

Reply:
Agreed, but this was kept consistent with the current approach in this file. Happy to amend this one if you'd like, and I can fix other instances in this file while I'm there too.

```js
app.post('/bootstrap', (req, res) => {                                          
  let { error, token } = bootstrap(req.body.password);                          
                                                                                
  if (error) {                                                                  
    res.status(400).send({ status: 'error', reason: error });                   
    return;                                                                     
  } else {                                                                      
    res.send({ status: 'ok', data: { token } });                                
  }                                                                             
});
```

Reply:
Yea this repo isn't the cleanest.. :) 

I primarily focus on the new additions and try to get those as clean as possible. So wouldn't want to block you on that. But if you have time to also amend the existing code - that would be amazing!

Reply:
Of course! I'd be happy to run through and do those others if you give me a couple of minutes :)